gen3-integration-tests/tests/test_presigned_url.py

```diff
@@ -0,0 +1,256 @@
+            raise
+
+    def test_get_presigned_url_with_protocol_not_exist_for_file(self):
+        """
+        Scenario: Get presigned-url with protocol not exist for file
+        Steps:
+            1. Create indexd record using http_link key information in files as setup
+            2. Create a signed url using did from step 1 and protocol as s3.
+            3. Verify the fence error is 'File {did} does not have a location with specified protocol s3.'
+        """
+        indexd_record = indexd_files["http_link"]
+        signed_url_res = self.fence.create_signed_url(
+            id=indexd_record["did"],
+            user="main_account",
+            expectedStatus=404,
+            params=["protocol=s3"],
+        )
+        msg = f"File {indexd_record['did']} does not have a location with specified protocol s3."
+        if msg not in signed_url_res.content.decode():
+            logger.error(f"{msg} not found")
+            logger.error(signed_url_res.content.decode())
+            raise
+
+    def test_get_presigned_url_no_data(self):
+        """
+        Scenario: Get presigned-url no data
+        Steps:
+            1. Create indexd record using no_link key information in files as setup
+            2. Create a signed url using did from step 1 and protocol as s3.
+            3. Verify the fence error is 'File {did} does not have a location with specified protocol s3.'
+        """
+        indexd_record = indexd_files["no_link"]
+        signed_url_res = self.fence.create_signed_url(
+            id=indexd_record["did"],
+            user="main_account",
+            expectedStatus=404,
+            params=["protocol=s3"],
+        )
+        msg = f"File {indexd_record['did']} does not have a location with specified protocol s3."
+        if msg not in signed_url_res.content.decode():
+            logger.error(f"{msg} not found")
+            logger.error(signed_url_res.content.decode())
+            raise
+
+    def test_get_presigned_url_no_requested_protocol_no_data(self):
+        """
+        Scenario: Get presigned-url no requested protocol, no data
+        Steps:
+            1. Create indexd record using no_link key information in files as setup
+            2. Create a signed url using did from step 1.
+            3. Verify the fence error is 'Can\'t find any file locations.'
+        """
+        indexd_record = indexd_files["no_link"]
+        signed_url_res = self.fence.create_signed_url(
+            id=indexd_record["did"], user="main_account", expectedStatus=404
+        )
+        msg = "Can&#39;t find any file locations."
+        if msg not in signed_url_res.content.decode():
+            logger.error(f"{msg} not found")
+            logger.error(signed_url_res.content.decode())
+            raise
+
+    @pytest.mark.clever_safe
+    def test_simple_clever_safe_presigner_url(self):
+        """
+        Scenario: Simple CleverSafe PreSigned URL test
```

Comment:
Maybe the test was written with the assumption that cleversafe is s3 api compliant so an S3 bucket was treated as a cleversafe one 🤔 ... but maybe we can delete this since this does not test anything different

Reply:
in gen3-qa the annotation was used for 
```
runTestsIfServiceVersion "@cleverSafe" "fence" "4.22.4" "2020.09"
```
but is not relevant now I guess

Reply:
right, if there is no cleversafe backend that the tests connect to, i don't think we need to keep this test